_plugins/author.rb

```diff
@@ -39,7 +39,7 @@ def render(context)
       full_name = context.environments[0]['page']['full_name']
       first_name = context.environments[0]['page']['first_name']
       posts = context.environments[0]['site']['posts']
-      site_url = context.environments[0]['site']['url']
+      site_url = context.environments[0]['site']['baseurl']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.